File: examples/v1/approval-rules/go/main.go
package main

import (
	v1 "github.com/pulumi/pulumi-pulumiservice/sdk/go/pulumiservice/v1"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		cfg := config.New(ctx, "")
		organizationName := cfg.Get("organizationName")
		if organizationName == "" {
			organizationName = "service-provider-test-org"
		}

		approvers, err := v1.NewPolicyGroup(ctx, "approvers", &v1.PolicyGroupArgs{
			OrgName:    pulumi.String(organizationName),
			Name:       pulumi.String("v1-approvers"),
			EntityType: pulumi.String("stacks"),
		})
		if err != nil {
			return err
		}

		ctx.Export("policyGroupName", approvers.Name)
		return nil
	})
}

File: examples/v1/approval-rules/python/__main__.py
import pulumi
import pulumi_pulumiservice.v1 as ps_v1

config = pulumi.Config()
organization_name = config.get("organizationName") or "service-provider-test-org"

approvers = ps_v1.PolicyGroup(
    "approvers",
    org_name=organization_name,
    name="v1-approvers",
    entity_type="stacks",
)

pulumi.export("policyGroupName", approvers.name)
